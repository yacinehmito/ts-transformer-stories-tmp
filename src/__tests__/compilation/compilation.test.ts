import { compile, extractArtifacts, DirectoryContent } from './compilation';

jest.setTimeout(15000);

expect.addSnapshotSerializer(DirectoryContent.getSnapshotSerializer());

test('The React project compiles as expected', async () => {
  await compile('react');
  expect(await extractArtifacts('react')).toMatchSnapshot();
});
