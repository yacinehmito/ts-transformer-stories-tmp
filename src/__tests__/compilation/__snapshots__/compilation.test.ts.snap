// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The React project compiles as expected 1`] = `
----boundary foo.stories.js
import { createElement } from 'react';
import { storiesOf as storiesOf_1 } from "@storybook/react";
const stories = storiesOf_1("Unnamed", module);
stories.add('Story in foo.stories.ts', () => createElement('div', null, 'Hello'));

----boundary index.stories.js
import { createElement } from 'react';
import { storiesOf as storiesOf_1 } from "@storybook/react";
const stories = storiesOf_1("Unnamed", module);
stories.add('Story in index.stories.ts', () => createElement('div', null, 'Hello'));

----boundary foo/foo.stories.js
import { createElement } from 'react';
import { storiesOf as storiesOf_1 } from "@storybook/react";
const stories = storiesOf_1("Unnamed", module);
stories.add('Story in foo/foo.stories.ts', () => createElement('div', null, 'Hello'));

----boundary foo/index.stories.js
import { createElement } from 'react';
import { storiesOf as storiesOf_1 } from "@storybook/react";
const stories = storiesOf_1("Unnamed", module);
stories.add('Story in foo/index.stories.ts', () => createElement('div', null, 'Hello'));

----boundary foo/bar/baz.stories.js
import { createElement } from 'react';
import { storiesOf as storiesOf_1 } from "@storybook/react";
const stories = storiesOf_1("Unnamed", module);
stories.add('Story in foo/bar/baz.stories.ts', () => createElement('div', null, 'Hello'));

`;
